src/ipfsd-daemon.js

```diff
@@ -128,24 +130,32 @@ class Daemon {
       this.path = initOpts.directory
     }
 
-    const args = ['init', '-b', initOpts.keysize || 2048]
+    const bits = initOpts.bits ? initOpts.bits : this.bits
+    const args = ['init']
+    // do not just set a default keysize,
+    // in case we decide to change it at
+    // the daemon level in the future
```

Comment:
This sentence doesn't read right. Do you mean "Skip if default daemon keysize"?

Reply:
Currently, ipfs daemons have a default keysize when using `ipfs init` - if we pass a value with `ipfs init -b <keysize>`, we're overriding that default (we're currently doing that in latest master). I don't think that is correct and we should rely on the default used by the daemon internally.